data/repositories.xml

```diff
@@ -16,5 +16,8 @@
     <phar alias="phpunit">
         <repository url="https://phar.phpunit.de/phive.xml"/>
     </phar>
+    <phar alias="phpbu">
+        <repository url="http://phar.phpbu.de/phive.xml"/>
```

Comment:
Maybe late, but: This is an HTTP url. For hopefully obvious reasons phive only supports HTTPS.
Please upgrade your infrastructure to use HTTPS.